src/interfaces/ARDUINO/ESPNOWHelper.h

```diff
@@ -208,11 +157,11 @@ class ENHelper {
 		uint8_t packet[PJON_PACKET_MAX_LENGTH + ESPNOW_PACKET_HEADER_LENGTH + ESPNOW_PACKET_FOOTER_LENGTH];
 
 		uint8_t len = length;
-		packet[0] = _magic_header[0] ^ len;
-		packet[1] = _magic_header[1] ^ len;
+		packet[0] = ESP_NOW_MAGIC_HEADER[0] ^ len;
+		packet[1] = ESP_NOW_MAGIC_HEADER[1] ^ len;
 		memcpy(packet + ESPNOW_PACKET_HEADER_LENGTH, data, len);
-		packet[len + 2] = _magic_header[2] ^ len;
-		packet[len + 3] = _magic_header[3] ^ len;
+		packet[len + 2] = ESP_NOW_MAGIC_HEADER[2] ^ len;
+		packet[len + 3] = ESP_NOW_MAGIC_HEADER[3] ^ len;
 
 		ATOMIC()
```

Comment:
Consider using `interrupts` and `noInterrupts` instead of the `ATOMIC` dependency, ie
```
noInterrupts();
Serial.println("Inside critical section.");
interrupts();
```

Reply:
Thanks! Fixed :) The dependency is removed.